addons/binding/org.openhab.binding.kodi/src/main/java/org/openhab/binding/kodi/protocol/KodiClientSocket.java

```diff
@@ -0,0 +1,227 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.kodi.protocol;
```

Comment:
This should be an internal package, e.g. org.openhab.binding.kodi.internal.protocol